<?xml version="1.0"?>
<launch>

<arg name="map" default="/put/your/mesh/map/path/here.ply" description="path to map file" />
<arg name="config" default="$(find-pkg-share rmcl)/config/micp_examples/hilti/uzh_tracking_area.yaml" description="path to config file" />

<node pkg="rmcl" exec="micp_localization" name="micp_localization" output="screen">
    <param name="map_file" value="$(var map)" />
    <param from="$(var config)" />
    <!-- remap default topic to topic where the initial pose is published -->
    <remap from="pose_wc" to="/initialpose" />
</node>

</launch>
